tests/bundle/Request/ParamConverter/SuggestionQueryParamConverterTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Bundle\Search\Request\ParamConverter;
+
+use Ibexa\Bundle\Search\Request\ParamConverter\SuggestionQueryParamConverter;
+use Ibexa\Search\Model\SuggestionQuery;
+use PHPUnit\Framework\TestCase;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Symfony\Component\HttpFoundation\Request;
+
+final class SuggestionQueryParamConverterTest extends TestCase
+{
+    private const DEFAULT_LIMIT = 10;
+
+    private SuggestionQueryParamConverter $converter;
+
+    protected function setUp(): void
+    {
+        $this->converter = new SuggestionQueryParamConverter(self::DEFAULT_LIMIT);
+    }
+
+    /**
+     * @dataProvider provideSupportsTestData
+     */
+    public function testSupports(string $class, bool $expectedResult): void
+    {
+        $configuration = new ParamConverter([]);
+        $configuration->setClass($class);
+
+        $this->assertSame($expectedResult, $this->converter->supports($configuration));
+    }
+
+    /**
+     * @return array<string, array{string, bool}>
+     */
+    public function provideSupportsTestData(): array
+    {
+        return [
+            'Supports SuggestionQuery' => [SuggestionQuery::class, true],
+            'Does not support other classes' => [\stdClass::class, false],
```

Comment:
```suggestion
            'Does not support other classes' => [stdClass::class, false],
```
AFAIK we prefer an import